resources/assets/js/components/PackageDetailFrame.vue

```diff
@@ -666,11 +663,50 @@ export default {
                     alert('Error: ' + response.message);
                 }
             );
-        }
-    },
+        },
+
+        fallbackCopyTextToClipboard() {
+            /* Get the text field */
+            var textArea = document.createElement("textarea");
+            textArea.value = this.composerString;
+
+            // Avoid scrolling to bottom
+            textArea.style.top = "0";
+            textArea.style.left = "0";
+            textArea.style.position = "fixed";
+
+            document.body.appendChild(textArea);
+            textArea.focus();
+            textArea.select();
+
+            try {
+                var successful = document.execCommand('copy');
+                var msg = successful ? 'successful' : 'unsuccessful';
+                console.log('Fallback: Copying text command was ' + msg);
+            } catch (err) {
+                console.error('Fallback: Oops, unable to copy', err);
+                return;
+            }
+
+            document.body.removeChild(textArea);
 
-    mounted() {
-        new ClipboardJS('#copy-button');
+            this.copySuccessful();
+        },
+
+        copyTextToClipboard() {
+            if (!navigator.clipboard) {
+                fallbackCopyTextToClipboard();
```

Comment:
I think this should be `this.fallbackCopyTextToClipboard();`?
```suggestion
                this.fallbackCopyTextToClipboard();
```